diff -urN subscription-manager-1.20.10.orig/cockpit/src/subscriptions-client.js subscription-manager-1.20.10/cockpit/src/subscriptions-client.js
--- subscription-manager-1.20.10.orig/cockpit/src/subscriptions-client.js	2022-09-28 14:44:30.903175268 +0200
+++ subscription-manager-1.20.10/cockpit/src/subscriptions-client.js	2022-09-28 14:45:34.120473232 +0200
@@ -46,7 +46,7 @@
 };
 
 const RHSM_DEFAULTS = { // TODO get these from a d-bus service instead
-    hostname: 'subscription.rhsm.redhat.com',
+    hostname: 'localhost',
     port: '443',
     prefix: '/subscription',
     proxy_port: '3128',
diff -urN subscription-manager-1.20.10.orig/etc-conf/rhsm.conf subscription-manager-1.20.10/etc-conf/rhsm.conf
--- subscription-manager-1.20.10.orig/etc-conf/rhsm.conf	2022-09-28 14:44:30.904175208 +0200
+++ subscription-manager-1.20.10/etc-conf/rhsm.conf	2022-09-28 14:45:34.120473232 +0200
@@ -3,7 +3,7 @@
 # Unified Entitlement Platform Configuration
 [server]
 # Server hostname:
-hostname = subscription.rhsm.redhat.com
+hostname = localhost
 
 # Server prefix:
 prefix = /subscription
diff -urN subscription-manager-1.20.10.orig/man/asciidoc/rhsm.conf.5.asciidoc subscription-manager-1.20.10/man/asciidoc/rhsm.conf.5.asciidoc
--- subscription-manager-1.20.10.orig/man/asciidoc/rhsm.conf.5.asciidoc	2022-09-28 14:44:30.915174780 +0200
+++ subscription-manager-1.20.10/man/asciidoc/rhsm.conf.5.asciidoc	2022-09-28 14:45:34.133473293 +0200
@@ -20,7 +20,7 @@
 ----------------
 hostname::
   The hostname of the subscription service being used. The default is the
-  Red Hat Customer Portal which is subscription.rhsm.redhat.com.
+  Red Hat Customer Portal which is localhost.
   This default should not be retrofitted to previously installed versions.
   It should be incorporated as the default going forward.
 
diff -urN subscription-manager-1.20.10.orig/man/subscription-manager.8 subscription-manager-1.20.10/man/subscription-manager.8
--- subscription-manager-1.20.10.orig/man/subscription-manager.8	2022-09-28 14:44:30.914174806 +0200
+++ subscription-manager-1.20.10/man/subscription-manager.8	2022-09-28 14:45:34.133473293 +0200
@@ -169,7 +169,7 @@
 .TP
 .B --serverurl=SERVER_HOSTNAME
 Passes the name of the subscription service with which to register the system. The default value, if this is not given, is the Customer Portal Subscription Management service,
-.B subscription.rhsm.redhat.com.
+.B localhost.
 If there is an on-premise subscription service such as Subscription Asset Manager, this parameter can be used to submit the hostname of the subscription service. For Subscription Asset Manager, if the Subscription Manager tool is configured with the Subscription Asset Manager RPM, then the default value for the
 .B --serverurl
 parameter is for the on-premise Subscription Asset Manager server.
@@ -544,7 +544,7 @@
 .TP
 .B --serverurl=SERVER_HOSTNAME
 Passes the name of the subscription service to use to list all available organizations. The \fBorgs\fP command will list all organizations for the specified service for which the user account is granted access. The default value, if this is not given, is the Customer Portal Subscription Management service,
-.B https://subscription.rhsm.redhat.com:443.
+.B https://localhost:443.
 If there is an on-premise subscription service such as Subscription Asset Manager, this parameter can be used to submit the hostname of the subscription service, in the form \fI[protocol://]servername[:port][/prefix]\fP. For Subscription Asset Manager, if the Subscription Manager tool is configured with the Subscription Asset Manager RPM, then the default value for the
 .B --serverurl
 parameter is for the on-premise Subscription Asset Manager server.
@@ -1227,7 +1227,7 @@
 The
 .B subscription-manager
 tool uses outgoing HTTPS requests. In the default configuration it will use HTTPS on port 443 to the subscription servers
-.B subscription.rhsm.redhat.com
+.B localhost
 and to the content delivery service
 .B cdn.redhat.com.
 
diff -urN subscription-manager-1.20.10.orig/README.md subscription-manager-1.20.10/README.md
--- subscription-manager-1.20.10.orig/README.md	2022-09-28 14:44:30.902175328 +0200
+++ subscription-manager-1.20.10/README.md	2022-09-28 14:45:34.118473222 +0200
@@ -51,7 +51,7 @@
 hostname and port are left alone (i.e. whatever is in the VM's `rhsm.conf` will
 be untouched).
 
-To register against subscription.rhsm.redhat.com, `.bashrc` might contain:
+To register against localhost, `.bashrc` might contain:
 ```bash
 export SUBMAN_RHSM_USERNAME=foobar
 export SUBMAN_RHSM_PASSWORD=password
diff -urN subscription-manager-1.20.10.orig/src/rhsm/config.py subscription-manager-1.20.10/src/rhsm/config.py
--- subscription-manager-1.20.10.orig/src/rhsm/config.py	2022-09-28 14:44:30.934174487 +0200
+++ subscription-manager-1.20.10/src/rhsm/config.py	2022-09-28 14:45:34.146473354 +0200
@@ -33,7 +33,7 @@
 
 # Defaults for connecting to RHSM, used to "reset" the configuration file
 # if requested by the user:
-DEFAULT_HOSTNAME = "subscription.rhsm.redhat.com"
+DEFAULT_HOSTNAME = "localhost"
 DEFAULT_PORT = "443"
 DEFAULT_PREFIX = "/subscription"
 
diff -urN subscription-manager-1.20.10.orig/subscription-manager.spec subscription-manager-1.20.10/subscription-manager.spec
--- subscription-manager-1.20.10.orig/subscription-manager.spec	2022-09-28 14:44:30.913174834 +0200
+++ subscription-manager-1.20.10/subscription-manager.spec	2022-09-28 14:45:34.130473278 +0200
@@ -1505,7 +1505,7 @@
 * Fri Dec 04 2015 Alex Wood <awood@redhat.com> 1.16.6-1
 - 1285004: Adds check for access to the required manager capabilty
   (csnyder@redhat.com)
-- 1278472: Change default registration url to subscription.rhsm.redhat.com
+- 1278472: Change default registration url to localhost
   (wpoteat@redhat.com)
 - 1275179: Do not allow quantity with auto attach (wpoteat@redhat.com)
 - 976859: Only check server version if asked. (alikins@redhat.com)
diff -urN subscription-manager-1.20.10.orig/test/rhsm/unit/config-tests.py subscription-manager-1.20.10/test/rhsm/unit/config-tests.py
--- subscription-manager-1.20.10.orig/test/rhsm/unit/config-tests.py	2022-09-28 14:44:30.919174696 +0200
+++ subscription-manager-1.20.10/test/rhsm/unit/config-tests.py	2022-09-28 14:45:34.136473306 +0200
@@ -278,7 +278,7 @@
         self.assertFalse(value)
 
     def test_is_default_true(self):
-        value = self.cfgParser.is_default('server', 'hostname', 'subscription.rhsm.redhat.com')
+        value = self.cfgParser.is_default('server', 'hostname', 'localhost')
         self.assertTrue(value)
 
     def test_is_default_false(self):
diff -urN subscription-manager-1.20.10.orig/test/rhsm/unit/util-tests.py subscription-manager-1.20.10/test/rhsm/unit/util-tests.py
--- subscription-manager-1.20.10.orig/test/rhsm/unit/util-tests.py	2022-09-28 14:44:30.919174696 +0200
+++ subscription-manager-1.20.10/test/rhsm/unit/util-tests.py	2022-09-28 14:45:34.135473302 +0200
@@ -85,9 +85,9 @@
 
     def test_hostname_with_scheme(self):
         # this is the default, so test it here
-        local_url = "https://subscription.rhsm.redhat.com/subscription"
+        local_url = "https://localhost/subscription"
         (username, password, hostname, port, prefix) = parse_url(local_url)
-        self.assertEqual("subscription.rhsm.redhat.com", hostname)
+        self.assertEqual("localhost", hostname)
         self.assertEqual(None, port)
         self.assertEqual("/subscription", prefix)
 
diff -urN subscription-manager-1.20.10.orig/test/test_managercli.py subscription-manager-1.20.10/test/test_managercli.py
--- subscription-manager-1.20.10.orig/test/test_managercli.py	2022-09-28 14:44:30.919174696 +0200
+++ subscription-manager-1.20.10/test/test_managercli.py	2022-09-28 14:45:34.140473326 +0200
@@ -299,7 +299,7 @@
     command_class = managercli.OwnersCommand
 
     def test_main_server_url(self):
-        server_url = "https://subscription.rhsm.redhat.com/subscription"
+        server_url = "https://localhost/subscription"
         self.cc.main(["--serverurl", server_url])
 
     def test_insecure(self):
@@ -310,7 +310,7 @@
     command_class = managercli.EnvironmentsCommand
 
     def test_main_server_url(self):
-        server_url = "https://subscription.rhsm.redhat.com/subscription"
+        server_url = "https://localhost/subscription"
         self.cc.main(["--serverurl", server_url])
 
     def test_insecure(self):
@@ -383,7 +383,7 @@
 
     def test_main_server_url(self):
         with patch.object(self.mock_cfg_parser, "save") as mock_save:
-            server_url = "https://subscription.rhsm.redhat.com/subscription"
+            server_url = "https://localhost/subscription"
             self._test_no_exception(["--serverurl", server_url])
             mock_save.assert_called_with()
 
@@ -1216,7 +1216,7 @@
         self.cc.cp = StubUEP()
 
     def test_main_server_url(self):
-        server_url = "https://subscription.rhsm.redhat.com/subscription"
+        server_url = "https://localhost/subscription"
         self.cc.main(["--serverurl", server_url])
 
     def test_insecure(self):
diff -urN subscription-manager-1.20.10.orig/test/test_migration.py subscription-manager-1.20.10/test/test_migration.py
--- subscription-manager-1.20.10.orig/test/test_migration.py	2022-09-28 14:44:30.920174681 +0200
+++ subscription-manager-1.20.10/test/test_migration.py	2022-09-28 14:45:34.145473349 +0200
@@ -239,7 +239,7 @@
 
     @patch.object(stubs.StubConfig, "get", autospec=True)
     def test_is_hosted(self, mock_get):
-        mock_get.return_value = "subscription.rhsm.redhat.com"
+        mock_get.return_value = "localhost"
         self.assertTrue(migrate.is_hosted())
         mock_get.return_value = "subscription.rhn.redhat.com"
         self.assertTrue(migrate.is_hosted())
diff -urN subscription-manager-1.20.10.orig/test/test_registrationgui.py subscription-manager-1.20.10/test/test_registrationgui.py
--- subscription-manager-1.20.10.orig/test/test_registrationgui.py	2022-09-28 14:44:30.918174714 +0200
+++ subscription-manager-1.20.10/test/test_registrationgui.py	2022-09-28 14:45:34.134473298 +0200
@@ -198,19 +198,19 @@
         self.assertTrue(self.screen.activation_key_checkbox.get_property('sensitive'))
 
     def test_activation_key_checkbox_prod_sensitive(self):
-        self.screen.server_entry.set_text("subscription.rhsm.redhat.com:443/baz")
+        self.screen.server_entry.set_text("localhost:443/baz")
         self.assertTrue(self.screen.activation_key_checkbox.get_property('sensitive'))
 
     def test_activation_key_checkbox_inactive_when_insensitive(self):
         self.screen.server_entry.set_text("foo.bar:443/baz")
         self.screen.activation_key_checkbox.set_active(True)
-        self.screen.server_entry.set_text("subscription.rhsm.redhat.com:443/baz")
+        self.screen.server_entry.set_text("localhost:443/baz")
         self.assertTrue(self.screen.activation_key_checkbox.get_property('sensitive'))
         self.assertTrue(self.screen.activation_key_checkbox.get_property('active'))
 
     @patch('subscription_manager.gui.registergui.config')
     def test__on_default_button_clicked(self, config):
-        config.DEFAULT_HOSTNAME = "subscription.rhsm.redhat.com"
+        config.DEFAULT_HOSTNAME = "localhost"
         config.DEFAULT_PORT = '443'
         config.DEFAULT_PREFIX = "/subscription"
 
diff -urN subscription-manager-1.20.10.orig/test/test_utils.py subscription-manager-1.20.10/test/test_utils.py
--- subscription-manager-1.20.10.orig/test/test_utils.py	2022-09-28 14:44:30.919174696 +0200
+++ subscription-manager-1.20.10/test/test_utils.py	2022-09-28 14:45:34.142473336 +0200
@@ -117,9 +117,9 @@
 
     def test_hostname_with_scheme(self):
         # this is the default, so test it here
-        local_url = "https://subscription.rhsm.redhat.com/subscription"
+        local_url = "https://localhost/subscription"
         (hostname, port, prefix) = parse_server_info(local_url)
-        self.assertEqual("subscription.rhsm.redhat.com", hostname)
+        self.assertEqual("localhost", hostname)
         self.assertEqual(DEFAULT_PORT, port)
         self.assertEqual("/subscription", prefix)
 
