diff '--color=auto' -ur a/m4/guestfs-appliance.m4 b/m4/guestfs-appliance.m4
--- a/m4/guestfs-appliance.m4	2022-05-26 15:56:51.388808404 +0200
+++ b/m4/guestfs-appliance.m4	2022-11-21 16:36:12.202126045 +0100
@@ -113,7 +113,7 @@
                   fi ) | tr '@<:@:lower:@:>@' '@<:@:upper:@:>@'
             )"
             AS_CASE([$DISTRO],
-                    [FEDORA | RHEL | CENTOS | ALMALINUX | CLOUDLINUX | ROCKY],
+                    [FEDORA | RHEL | CENTOS | ALMALINUX | CLOUDLINUX | ROCKY | EUROLINUX],
                         [DISTRO=REDHAT],
                     [OPENSUSE* | SLED | SLES],[DISTRO=SUSE],
                     [ARCH | MANJARO | ARTIX],[DISTRO=ARCHLINUX],
