From 0464323da7ad62aefa0f2880930ad131ebe20a74 Mon Sep 17 00:00:00 2001
From: Alex Baranowski <alex@euro-linux.com>
Date: Tue, 7 Nov 2023 15:50:11 +0000
Subject: [PATCH] enable eurolinux

---
 m4/guestfs-appliance.m4 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/m4/guestfs-appliance.m4 b/m4/guestfs-appliance.m4
index 4e671d2..507850e 100644
--- a/m4/guestfs-appliance.m4
+++ b/m4/guestfs-appliance.m4
@@ -114,7 +114,7 @@ if test "x$ENABLE_APPLIANCE" = "xyes"; then
                       fi ) | tr '@<:@:lower:@:>@' '@<:@:upper:@:>@'
                 )"
                 AS_CASE([$DISTRO],
-                        [FEDORA | RHEL | CENTOS | ALMALINUX | CLOUDLINUX \
+                        [FEDORA | EUROLINUX | RHEL | CENTOS | ALMALINUX | CLOUDLINUX \
 			 | ROCKY | VIRTUOZZO],
                             [DISTRO=REDHAT],
                         [OPENSUSE* | SLED | SLES],[DISTRO=SUSE],
-- 
2.39.3

