--- abrt.spec.orig	2022-09-09 13:03:31.762610536 +0200
+++ abrt.spec	2022-09-09 13:03:54.408762960 +0200
@@ -472,7 +472,7 @@
 Requires: libreport-cli
 Requires: abrt-addon-kerneloops
 Requires: abrt-addon-ccpp, abrt-addon-python
-Requires: libreport-plugin-logger, libreport-plugin-rhtsupport, libreport-plugin-mailx
+Requires: libreport-plugin-logger, libreport-plugin-mailx
 Requires: sos
 
 %description cli
@@ -492,7 +492,7 @@
 # Default config of addon-ccpp requires gdb
 Requires: gdb >= 7.0-3
 Requires: abrt-gui
-Requires: libreport-plugin-logger, libreport-plugin-rhtsupport, libreport-plugin-mailx
+Requires: libreport-plugin-logger, libreport-plugin-mailx
 Requires: sos
 #Requires: abrt-plugin-firefox
 %if 0%{?fedora}
