diff --git a/postgis34.spec b/postgis34.spec
index 828252f..0719e0e 100644
--- a/postgis34.spec
+++ b/postgis34.spec
@@ -199,6 +199,7 @@ This packages provides JIT support for postgis33
 %{__cp} -p %{SOURCE2} %{sname}-%{version}.pdf

 %build
+CC="gcc"; export CC
 LDFLAGS="-Wl,-rpath,%{geosinstdir}/lib64 ${LDFLAGS}" ; export LDFLAGS
 LDFLAGS="-Wl,-rpath,%{projinstdir}/lib ${LDFLAGS}" ; export LDFLAGS
 SHLIB_LINK="$SHLIB_LINK -Wl,-rpath,%{geosinstdir}/lib64" ; export SHLIB_LINK
