--- chrony.spec	2022-07-14 07:26:27.000000000 +0000
+++ chrony-fixed.spec	2022-10-05 21:38:03.899207337 +0000
@@ -59,9 +59,9 @@
 can also operate as an NTPv4 (RFC 5905) server and peer to provide a time
 service to other computers in the network.
 
-%if 0%{!?vendorzone:1}
-%global vendorzone %(source /etc/os-release && echo ${ID}.)
-%endif
+#%if 0%{!?vendorzone:1}
+#%global vendorzone %(source /etc/os-release && echo ${ID}.)
+#%endif
 
 %prep
 %setup -q -n %{name}-%{version}%{?prerelease} -a 10
@@ -86,14 +86,13 @@
 EOF
 
 # don't allow packaging without vendor zone
-test -n "%{vendorzone}"
+#test -n "%{vendorzone}"
 
 # use example chrony.conf as the default config with some modifications:
 # - use our vendor zone (2.*pool.ntp.org names include IPv6 addresses)
 # - enable leapsectz to get TAI-UTC offset and leap seconds from tzdata
 # - enable keyfile
-sed -e 's|^\(pool \)\(pool.ntp.org\)|\12.%{vendorzone}\2|' \
-    -e 's|#\(leapsectz\)|\1|' \
+sed -e 's|#\(leapsectz\)|\1|' \
     -e 's|#\(keyfile\)|\1|' \
         < examples/chrony.conf.example2 > chrony.conf
 
