--- ../rhel/osinfo-db.spec	2021-12-16 11:33:00.000000000 +0100
+++ osinfo-db.spec	2022-08-24 12:02:25.415289636 +0200
@@ -10,6 +10,8 @@
 URL: http://libosinfo.org/
 BuildRequires: intltool
 BuildRequires: osinfo-db-tools
+# EuroLinux patch requires git
+BuildRequires: git
 BuildArch: noarch
 Requires: hwdata
 
@@ -20,6 +22,11 @@
 
 %install
 osinfo-db-import  --root %{buildroot} --dir %{_datadir}/osinfo %{SOURCE0}
+
+cd %{buildroot}/%{_datadir}/osinfo
+git apply %{SOURCE100}
+cd -
+
 %if 0%{?rhel}
 # Remove the upstream virtio-win / spice-guest-tools drivers
 find %{buildroot}/%{_datadir}/osinfo/os/microsoft.com/ -name "win-*.d" -type d -exec rm -rf {} +

