diff --git a/postgis34.spec b/postgis34.spec
index 798c61a..ac32519 100644
--- a/postgis34.spec
+++ b/postgis34.spec
@@ -211,7 +211,7 @@ export PKG_CONFIG_PATH=$PKG_CONFIG_PATH:%{projinstdir}/lib64/pkgconfig

 autoconf

-%configure --with-pgconfig=%{pginstdir}/bin/pg_config \
+%configure CC='gcc'  --with-pgconfig=%{pginstdir}/bin/pg_config \
        --bindir=%{pginstdir}/bin/ \
        --datadir=%{pginstdir}/share/ \
        --enable-lto \
