diff --git a/postgis34.spec b/postgis34.spec
index 798c61a..b55d694 100644
--- a/postgis34.spec
+++ b/postgis34.spec
@@ -4,8 +4,6 @@
 %global postgiscurrmajorversion %(echo %{postgismajorversion}|tr -d '.')
 %global sname  postgis

-%pgdg_set_gis_variables
-
 # Override some variables. PostGIS 3.4 is best served with GeOS 3.12,
 # GDAL 3.8 and PROJ 9.4:
 %global geosfullversion %geos312fullversion