diff --git a/postgis34.spec b/postgis34.spec
index 798c61a..2446f0f 100644
--- a/postgis34.spec
+++ b/postgis34.spec
@@ -205,13 +205,13 @@ LDFLAGS="-Wl,-rpath,%{projinstdir}/lib ${LDFLAGS}" ; export LDFLAGS
 SHLIB_LINK="$SHLIB_LINK -Wl,-rpath,%{geosinstdir}/lib64" ; export SHLIB_LINK
 SFCGAL_LDFLAGS="$SFCGAL_LDFLAGS -L/usr/lib64"; export SFCGAL_LDFLAGS

-LDFLAGS="$LDFLAGS -L%{geosinstdir}/lib64 -lgeos_c -L%{projinstdir}/lib -L%{gdalinstdir}/lib -L%{libgeotiffinstdir}/lib -ltiff -L/usr/lib64"; export LDFLAGS
-CFLAGS="$CFLAGS -I%{gdalinstdir}/include"; export CFLAGS
+LDFLAGS="$LDFLAGS -L%{geosinstdir}/lib64 -lgeos_c -L%{projinstdir}/lib -L%{gdalinstdir}/lib64 -L%{libgeotiffinstdir}/lib -ltiff -L/usr/lib64"; export LDFLAGS
+CFLAGS="$CFLAGS -I%{gdalinstdir}/include/gdal/"; export CFLAGS
 export PKG_CONFIG_PATH=$PKG_CONFIG_PATH:%{projinstdir}/lib64/pkgconfig

 autoconf

-%configure --with-pgconfig=%{pginstdir}/bin/pg_config \
+%configure CC='gcc'  --with-pgconfig=%{pginstdir}/bin/pg_config \
        --bindir=%{pginstdir}/bin/ \
        --datadir=%{pginstdir}/share/ \
        --enable-lto \
