--- /home/Alex/workspace/EuroLinux/python-rpmpatch/srpms/rhel/eth-tools.spec	2021-11-11 09:06:24.000000000 +0100
+++ /home/Alex/workspace/EuroLinux/python-rpmpatch/srpms/el/eth-tools.spec	2022-04-10 16:42:54.000000000 +0200
@@ -39,7 +42,7 @@
 Contains tools for managing fabric on a management node.
 
 %prep
-%autosetup -n eth-fast-fabric-%{version}
+%autosetup -n eth-fast-fabric-%{version} -p1
 
 %build
 cd OpenIb_Host
