--- /home/Alex/workspace/EuroLinux/python-rpmpatch/srpms/rhel/crash.spec	2022-02-08 05:14:11.000000000 +0100
+++ /home/Alex/workspace/EuroLinux/python-rpmpatch/srpms/el/crash.spec	2022-08-23 14:39:18.560775857 +0200
@@ -36,7 +36,7 @@
 %description
 The core analysis suite is a self-contained tool that can be used to
 investigate either live systems, kernel core dumps created from the
-netdump, diskdump and kdump packages from Red Hat Linux, the mcore kernel patch
+netdump, diskdump and kdump packages from EuroLinux, the mcore kernel patch
 offered by Mission Critical Linux, or the LKCD kernel patch.
 
 %package devel
@@ -47,7 +47,7 @@
 %description devel
 The core analysis suite is a self-contained tool that can be used to
 investigate either live systems, kernel core dumps created from the
-netdump, diskdump and kdump packages from Red Hat Linux, the mcore kernel patch
+netdump, diskdump and kdump packages from EuroLinux, the mcore kernel patch
 offered by Mission Critical Linux, or the LKCD kernel patch.
 
 %prep
