diff --git a/postgresql-16.spec b/postgresql-16.spec
index 93ebd0a..709d7d2 100644
--- a/postgresql-16.spec
+++ b/postgresql-16.spec
@@ -58,9 +58,9 @@ Version:      16.2
 %if 0%{?suse_version} >= 1315
 # SuSE upstream packages have release numbers like 150200.5.19.1
 # which overrides our packages. Increase our release number on SuSE.
-Release:       420001PGDG%{?dist}
+Release:       420001EuroDB%{?dist}
 %else
-Release:       1PGDG%{?dist}
+Release:       1EuroDB%{?dist}
 %endif
 License:       PostgreSQL
 Url:           https://www.postgresql.org/
