diff -uNr pip.og/pip/_vendor/distro.py pip.el/pip/_vendor/distro.py
--- pip.og/pip/_vendor/distro.py        2021-07-16 07:23:41.879691318 +0000
+++ pip.el/pip/_vendor/distro.py        2021-07-16 07:27:10.587691318 +0000
@@ -139,6 +139,7 @@
     "debian"        Debian
     "rhel"          RedHat Enterprise Linux
     "centos"        CentOS
+    "eurolinux"     EuroLinux
     "fedora"        Fedora
     "sles"          SUSE Linux Enterprise Server
     "opensuse"      openSUSE
