From c74a059958b4e28f4b647da2edfa7ff7243f627b Mon Sep 17 00:00:00 2001
From: Alex Baranowski <alex@euro-linux.com>
Date: Sun, 9 Apr 2023 19:16:13 +0000
Subject: [PATCH] eurolinux patch

---
 extras/source/wordbook/technical.dic | 8 ++++++++
 1 files changed, 8 insertions(+)

diff --git a/extras/source/wordbook/technical.dic b/extras/source/wordbook/technical.dic
index f2af65fad..51ddfead0 100644
--- a/extras/source/wordbook/technical.dic
+++ b/extras/source/wordbook/technical.dic
@@ -80,6 +80,14 @@ eGuide
 Electronics
 Etch
 Ethernet
+EuroAP=
+EuroDB=
+EuroIQ=
+EuroLinux=
+EuroMan=
+EuroSSO=
+EuroStorage=
+EuroTower=
 Evolution
 Exadata=
 Facebook
-- 
2.27.0

