--- ipa-4.6.8-5.sl7_9.14.src/ipa.spec	2023-05-16 03:01:03.000000000 +0000
+++ ipa-4.6.8-5.el7_9.14.src/ipa.spec	2023-05-29 06:27:03.198524864 +0000
@@ -974,10 +977,10 @@
 %endif
 # with_python3
 
-%{__cp} /usr/share/sl-logos/os-name-white.png install/ui/images/header-logo.png
+%{__cp} /usr/share/el-logos/os-name-white.png install/ui/images/header-logo.png
 %{__cp} /usr/share/backgrounds/Fixation-1.jpg install/ui/images/login-screen-background.jpg
 %{__cp} /usr/share/pixmaps/system-logo-white.png install/ui/images/login-screen-logo.png
-%{__cp} /usr/share/sl-logos/os-name-white-big.png install/ui/images/product-name.png
+%{__cp} /usr/share/el-logos/os-name-white-big.png install/ui/images/product-name.png
 
 %build
 # RHEL spec file only: START
