diff -uNr cloud-init-20.3/cloudinit/config/cc_ntp.py cloud-init-20.3p/cloudinit/config/cc_ntp.py
--- cloud-init-20.3/cloudinit/config/cc_ntp.py	2020-08-25 16:28:27.000000000 +0000
+++ cloud-init-20.3p/cloudinit/config/cc_ntp.py	2021-08-03 11:35:05.637615301 +0000
@@ -24,8 +24,9 @@
 frequency = PER_INSTANCE
 NTP_CONF = '/etc/ntp.conf'
 NR_POOL_SERVERS = 4
-distros = ['alpine', 'centos', 'debian', 'fedora', 'opensuse', 'rhel',
-           'sles', 'ubuntu']
+distros = ['almalinux', 'alpine', 'centos', 'debian', 'eurolinux', 'fedora',
+           'opensuse', 'photon', 'rhel', 'rocky', 'sles', 'ubuntu',
+           'virtuozzo']
 
 NTP_CLIENT_CONFIG = {
     'chrony': {
@@ -384,9 +385,9 @@
         # For legal reasons x.pool.sles.ntp.org does not exist,
         # use the opensuse pool
         pool_distro = 'opensuse'
-    elif distro == 'alpine':
+    elif distro == 'alpine' or distro == 'eurolinux':
         # Alpine-specific pool (i.e. x.alpine.pool.ntp.org) does not exist
-        # so use general x.pool.ntp.org instead.
+        # so use general x.pool.ntp.org instead. The same applies to EuroLinux
         pool_distro = ''
 
     for x in range(0, NR_POOL_SERVERS):
diff -uNr cloud-init-20.3/cloudinit/config/cc_yum_add_repo.py cloud-init-20.3p/cloudinit/config/cc_yum_add_repo.py
--- cloud-init-20.3/cloudinit/config/cc_yum_add_repo.py	2020-08-25 16:28:27.000000000 +0000
+++ cloud-init-20.3p/cloudinit/config/cc_yum_add_repo.py	2021-08-03 11:36:14.490615301 +0000
@@ -18,7 +18,8 @@
 
 **Module frequency:** per always
 
-**Supported distros:** centos, fedora, rhel
+**Supported distros:** almalinux, centos, eurolinux, fedora, photon, rhel,
+                       rocky, virtuozzo
 
 **Config keys**::
 
@@ -36,7 +37,8 @@
 
 from cloudinit import util
 
-distros = ['centos', 'fedora', 'rhel']
+distros = ['almalinux', 'centos', 'eurolinux', 'fedora', 'photon', 'rhel',
+           'rocky', 'virtuozzo']
 
 
 def _canonicalize_id(repo_id):
diff -uNr cloud-init-20.3/cloudinit/distros/eurolinux.py cloud-init-20.3p/cloudinit/distros/eurolinux.py
--- cloud-init-20.3/cloudinit/distros/eurolinux.py	1970-01-01 00:00:00.000000000 +0000
+++ cloud-init-20.3p/cloudinit/distros/eurolinux.py	2021-08-03 11:37:01.953615301 +0000
@@ -0,0 +1,9 @@
+# This file is part of cloud-init. See LICENSE file for license information.
+
+from cloudinit.distros import rhel
+
+
+class Distro(rhel.Distro):
+    pass
+
+# vi: ts=4 expandtab
diff -uNr cloud-init-20.3/cloudinit/distros/__init__.py cloud-init-20.3p/cloudinit/distros/__init__.py
--- cloud-init-20.3/cloudinit/distros/__init__.py	2020-08-25 16:28:27.000000000 +0000
+++ cloud-init-20.3p/cloudinit/distros/__init__.py	2021-08-03 11:36:36.629615301 +0000
@@ -45,7 +45,8 @@
     'debian': ['debian', 'ubuntu'],
     'freebsd': ['freebsd'],
     'gentoo': ['gentoo'],
-    'redhat': ['amazon', 'centos', 'fedora', 'rhel'],
+    'redhat': ['almalinux', 'amazon', 'centos', 'eurolinux', 'fedora',
+               'photon', 'rhel', 'rocky', 'virtuozzo'],
     'suse': ['opensuse', 'sles'],
 }
 
diff -uNr cloud-init-20.3/cloudinit/net/sysconfig.py cloud-init-20.3p/cloudinit/net/sysconfig.py
--- cloud-init-20.3/cloudinit/net/sysconfig.py	2020-08-25 16:28:27.000000000 +0000
+++ cloud-init-20.3p/cloudinit/net/sysconfig.py	2021-08-03 11:37:20.531615301 +0000
@@ -19,7 +19,8 @@
 
 LOG = logging.getLogger(__name__)
 NM_CFG_FILE = "/etc/NetworkManager/NetworkManager.conf"
-KNOWN_DISTROS = ['centos', 'fedora', 'rhel', 'suse']
+KNOWN_DISTROS = ['almalinux', 'centos', 'eurolinux', 'fedora', 'rhel', 'rocky',
+                 'suse', 'virtuozzo']
 
 
 def _make_header(sep='#'):
diff -uNr cloud-init-20.3/cloudinit/util.py cloud-init-20.3p/cloudinit/util.py
--- cloud-init-20.3/cloudinit/util.py	2020-08-25 16:28:27.000000000 +0000
+++ cloud-init-20.3p/cloudinit/util.py	2021-08-03 15:05:54.150106169 +0000
@@ -548,8 +548,8 @@
     if system == "linux":
         linux_dist = info['dist'][0].lower()
         if linux_dist in (
-                'alpine', 'arch', 'centos', 'debian', 'fedora', 'rhel',
-                'suse'):
+                'almalinux', 'alpine', 'arch', 'centos', 'debian', 'eurolinux',
+                'fedora', 'photon', 'rhel', 'rocky', 'suse', 'virtuozzo'):
             var = linux_dist
         elif linux_dist in ('ubuntu', 'linuxmint', 'mint'):
             var = 'ubuntu'
diff -uNr cloud-init-20.3/config/cloud.cfg.tmpl cloud-init-20.3p/config/cloud.cfg.tmpl
--- cloud-init-20.3/config/cloud.cfg.tmpl	2020-08-25 16:28:27.000000000 +0000
+++ cloud-init-20.3p/config/cloud.cfg.tmpl	2021-08-03 11:40:09.948615301 +0000
@@ -21,7 +21,8 @@
 disable_root: true
 {% endif %}
 
-{% if variant in ["alpine", "amazon", "centos", "fedora", "rhel"] %}
+{% if variant in ["almalinux", "alpine", "amazon", "centos", "eurolinux",
+                  "fedora", "rhel", "rocky", "virtuozzo"] %}
 mount_default_fields: [~, ~, 'auto', 'defaults,nofail', '0', '2']
 {% if variant == "amazon" %}
 resize_rootfs: noblock
@@ -152,8 +153,8 @@
 system_info:
    # This will affect which distro class gets used
 {% if variant in ["alpine", "amazon", "arch", "centos", "debian",
-                  "fedora", "freebsd", "netbsd", "openbsd", "rhel",
-                  "suse", "ubuntu"] %}
+                  "eurolinux", "fedora", "freebsd", "netbsd", "openbsd",
+                  "photon", "rhel", "rocky", "suse", "ubuntu", "virtuozzo"] %}
    distro: {{ variant }}
 {% else %}
    # Unknown/fallback distro.
@@ -204,8 +205,8 @@
          primary: http://ports.ubuntu.com/ubuntu-ports
          security: http://ports.ubuntu.com/ubuntu-ports
    ssh_svcname: ssh
-{% elif variant in ["alpine", "amazon", "arch", "centos", "fedora",
-                    "rhel", "suse"] %}
+{% elif variant in ["almalinux", "alpine", "amazon", "arch", "centos", "eurolinux",
+                    "fedora", "rhel", "rocky", "suse", "virtuozzo"] %}
    # Default user name + that default users groups (if added/used)
    default_user:
 {% if variant == "amazon" %}
diff -uNr cloud-init-20.3/systemd/cloud-init-generator.tmpl cloud-init-20.3p/systemd/cloud-init-generator.tmpl
--- cloud-init-20.3/systemd/cloud-init-generator.tmpl	2020-08-25 16:28:27.000000000 +0000
+++ cloud-init-20.3p/systemd/cloud-init-generator.tmpl	2021-08-03 11:40:43.700615301 +0000
@@ -83,7 +83,8 @@
 
 check_for_datasource() {
     local ds_rc=""
-{% if variant in ["rhel", "fedora", "centos"] %}
+{% if variant in ["almalinux", "centos", "eurolinux", "fedora", "rhel", 
+                  "rocky", "virtuozzo"] %}
     local dsidentify="/usr/libexec/cloud-init/ds-identify"
 {% else %}
     local dsidentify="/usr/lib/cloud-init/ds-identify"
diff -uNr cloud-init-20.3/systemd/cloud-init.service.tmpl cloud-init-20.3p/systemd/cloud-init.service.tmpl
--- cloud-init-20.3/systemd/cloud-init.service.tmpl	2020-08-25 16:28:27.000000000 +0000
+++ cloud-init-20.3p/systemd/cloud-init.service.tmpl	2021-08-03 11:41:07.822615301 +0000
@@ -10,7 +10,8 @@
 {% if variant in ["ubuntu", "unknown", "debian"] %}
 After=networking.service
 {% endif %}
-{% if variant in ["centos", "fedora", "rhel"] %}
+{% if variant in ["almalinux", "centos", "eurolinux", "fedora", "rhel", 
+                  "rocky", "virtuozzo"] %}
 After=network.service
 After=NetworkManager.service
 {% endif %}
diff -uNr cloud-init-20.3/tools/read-dependencies cloud-init-20.3p/tools/read-dependencies
--- cloud-init-20.3/tools/read-dependencies	2020-08-25 16:28:27.000000000 +0000
+++ cloud-init-20.3p/tools/read-dependencies	2021-08-03 11:43:29.657615301 +0000
@@ -24,6 +24,7 @@
 DISTRO_PKG_TYPE_MAP = {
     'centos': 'redhat',
     'redhat': 'redhat',
+    'eurolinux': 'redhat',
     'debian': 'debian',
     'ubuntu': 'debian',
     'opensuse': 'suse',
@@ -66,11 +67,13 @@
 DRY_DISTRO_INSTALL_PKG_CMD = {
     'centos': ['yum', 'install', '--assumeyes'],
     'redhat': ['yum', 'install', '--assumeyes'],
+    'eurolinux': ['yum', 'install', '--assumeyes'],
 }
 
 DISTRO_INSTALL_PKG_CMD = {
     'centos': MAYBE_RELIABLE_YUM_INSTALL,
     'redhat': MAYBE_RELIABLE_YUM_INSTALL,
+    'eurolinux': MAYBE_RELIABLE_YUM_INSTALL,
     'debian': ['apt', 'install', '-y'],
     'ubuntu': ['apt', 'install', '-y'],
     'opensuse': ZYPPER_INSTALL,
@@ -82,6 +85,7 @@
 CI_SYSTEM_BASE_PKGS = {
     'common': ['make', 'sudo', 'tar'],
     'redhat': ['python3-tox'],
+    'eurolinux': ['python3-tox'],
     'centos': ['python3-tox'],
     'ubuntu': ['devscripts', 'python3-dev', 'libssl-dev', 'tox', 'sbuild'],
     'debian': ['devscripts', 'python3-dev', 'libssl-dev', 'tox', 'sbuild']}
@@ -273,10 +277,10 @@
         cmd = DRY_DISTRO_INSTALL_PKG_CMD[distro]
     install_cmd.extend(cmd)
 
-    if distro in ['centos', 'redhat']:
+    if distro in ['centos', 'redhat', 'rocky', 'eurolinux']:
         # CentOS and Redhat need epel-release to access oauthlib and jsonschema
         subprocess.check_call(install_cmd + ['epel-release'])
-    if distro in ['suse', 'opensuse', 'redhat', 'centos']:
+    if distro in ['suse', 'opensuse', 'redhat', 'rocky', 'centos', 'eurolinux']:
         pkg_list.append('rpm-build')
     subprocess.check_call(install_cmd + pkg_list)
 
diff -uNr cloud-init-20.3/tools/render-cloudcfg cloud-init-20.3p/tools/render-cloudcfg
--- cloud-init-20.3/tools/render-cloudcfg	2020-08-25 16:28:27.000000000 +0000
+++ cloud-init-20.3p/tools/render-cloudcfg	2021-08-03 11:42:03.151615301 +0000
@@ -4,9 +4,9 @@
 import os
 import sys
 
-VARIANTS = ["alpine", "amazon", "arch", "centos", "debian", "fedora",
-            "freebsd", "netbsd", "openbsd", "rhel", "suse", "ubuntu",
-            "unknown"]
+VARIANTS = ["almalinux", "alpine", "amazon", "arch", "centos", "debian",
+            "eurolinux", "fedora", "freebsd", "netbsd", "openbsd", "photon",
+            "rhel", "suse","rocky", "ubuntu", "unknown", "virtuozzo"]
 
 
 if "avoid-pep8-E402-import-not-top-of-file":
