diff -ru --new-file sos-4.5.3/sos/policies/distros/eurolinux.py sos-4.5.3_EL/sos/policies/distros/eurolinux.py
--- sos-4.5.3/sos/policies/distros/eurolinux.py	1970-01-01 01:00:00.000000000 +0100
+++ sos-4.5.3_EL/sos/policies/distros/eurolinux.py	2023-06-01 15:25:05.219000000 +0200
@@ -0,0 +1,51 @@
+# Copyright (C) Paweł Piasek <pp@euro-linux.com>
+
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.policies.distros.redhat import RedHatPolicy, OS_RELEASE
+import os
+
+
+class EuroLinuxPolicy(RedHatPolicy):
+    distro = "EuroLinux"
+    vendor = "EuroLinux"
+    vendor_urls = [
+            ('Distribution Website', 'https://euro-linux.com'),
+            ('Commercial Support', 'https://support.euro-linux.com')
+    ]
+
+    def __init__(self, sysroot=None, init=None, probe_runtime=True,
+                 remote_exec=None):
+        super(EuroLinuxPolicy, self).__init__(sysroot=sysroot, init=init,
+                                          probe_runtime=probe_runtime,
+                                          remote_exec=remote_exec)
+
+    @classmethod
+    def check(cls, remote=''):
+        if remote:
+            return cls.distro in remote
+
+        # Return False if /etc/os-release is missing
+        if not os.path.exists(OS_RELEASE):
+            return False
+
+        # Return False if /etc/el-release is missing
+        if not os.path.isfile('/etc/el-release'):
+            return False
+
+        # If we've gotten this far, check for EuroLinux in
+        # /etc/os-release
+        with open(OS_RELEASE, 'r') as f:
+            for line in f:
+                if line.startswith('NAME'):
+                    if 'EuroLinux' in line:
+                        return True
+
+        return False
+
+# vim: set et ts=4 sw=4 :
