--- osinfo-db.spec	2022-11-22 15:34:15.794427473 +0100
+++ osinfo-db.spec_EL	2022-11-22 15:41:01.872423463 +0100
@@ -16,6 +16,8 @@
 BuildRequires: intltool
 BuildRequires: git-core
 BuildRequires: osinfo-db-tools
+# EuroLinux patch requires git
+BuildRequires: git
 BuildArch: noarch
 Requires: hwdata
 
@@ -33,6 +35,11 @@
 
 %install
 osinfo-db-import  --root %{buildroot} --dir %{_datadir}/osinfo %{PatchedSource}
+
+cd %{buildroot}/%{_datadir}/osinfo
+git apply %{SOURCE100}
+cd -
+
 %if 0%{?rhel}
 # Remove the upstream virtio-win / spice-guest-tools drivers
 find %{buildroot}/%{_datadir}/osinfo/os/microsoft.com/ -name "win-*.d" -type d -exec rm -rf {} +
