--- rhel/anaconda.spec	2022-03-09 15:27:56.000000000 +0100
+++ el/anaconda.spec	2022-08-24 13:15:26.155097320 +0200
@@ -265,6 +267,13 @@
 %prep
 %autosetup -p 1
 
+# Additonal debrand
+sed -e 's/RHEL/EuroLinux/' -i po/*.po
+sed -e 's/Red Hat Enterprise Linux/EuroLinux/' -i po/*.po
+
+# EuroLinux installation class/conf
+cp %{SOURCE1} data/product.d/
+
 %build
 # use actual build-time release number, not tarball creation time release number
 %configure ANACONDA_RELEASE=%{release}

