diff -uNr anaconda-33.16.3.26/data/anaconda-gtk.css anaconda-33.16.3.26p/data/anaconda-gtk.css
--- anaconda-33.16.3.26/data/anaconda-gtk.css	2020-08-04 17:17:51.000000000 +0200
+++ anaconda-33.16.3.26p/data/anaconda-gtk.css	2021-05-10 14:42:39.509457306 +0200
@@ -92,10 +92,13 @@
 
 @define-color redhat #2d2d2d;
 @define-color fedora #2f4265;
+/* EuroLinux Patch */
+@define-color eurolinux #0c61ab;
 
 /* theme colors/images */
 
-@define-color product_bg_color @redhat;
+/* EuroLinux Patch */
+@define-color product_bg_color @eurolinux;
 
 /* logo and sidebar classes */
 
diff -uNr anaconda-33.16.3.26/data/liveinst/gnome/rhel-welcome.desktop anaconda-33.16.3.26p/data/liveinst/gnome/rhel-welcome.desktop
--- anaconda-33.16.3.26/data/liveinst/gnome/rhel-welcome.desktop	2020-05-25 17:17:00.000000000 +0200
+++ anaconda-33.16.3.26p/data/liveinst/gnome/rhel-welcome.desktop	2021-05-10 14:46:43.790832125 +0200
@@ -1,5 +1,5 @@
 [Desktop Entry]
-Name=Welcome to Red Hat Enterprise Linux
+Name=Welcome to EuroLinux 
 Exec=/usr/share/anaconda/gnome/rhel-welcome
 Terminal=false
 Type=Application
diff -uNr anaconda-33.16.3.26/data/liveinst/gnome/rhel-welcome.js anaconda-33.16.3.26p/data/liveinst/gnome/rhel-welcome.js
--- anaconda-33.16.3.26/data/liveinst/gnome/rhel-welcome.js	2020-05-27 06:53:25.000000000 +0200
+++ anaconda-33.16.3.26p/data/liveinst/gnome/rhel-welcome.js	2021-05-10 15:05:55.458146082 +0200
@@ -59,7 +59,7 @@
                                                 default_width: 600,
                                                 default_height: 550,
                                                 skip_taskbar_hint: true,
-                                                title: _("Welcome to Red Hat Enterprise Linux"),
+                                                title: _("Welcome to EuroLinux"),
                                                 window_position: Gtk.WindowPosition.CENTER });
       this.window.connect('key-press-event', Lang.bind(this,
           function(w, event) {
@@ -104,7 +104,7 @@
       let installButton = new Gtk.Button({ child: installContent });
       buttonBox.add(installButton);
 
-      this._label = makeLabel(_("You are currently running Red Hat Enterprise Linux from live media.\nYou can install Red Hat Enterprise Linux now, or choose \"Install to Hard Drive\" in the Activities Overview at any later time."), false);
+      this._label = makeLabel(_("You are currently running EuroLinux from live media.\nYou can install EuroLinux now, or choose \"Install to Hard Drive\" in the Activities Overview at any later time."), false);
       mainGrid.add(this._label);
 
       installButton.connect('clicked', Lang.bind(this,
diff -uNr anaconda-33.16.3.26/docs/intro.rst anaconda-33.16.3.26p/docs/intro.rst
--- anaconda-33.16.3.26/docs/intro.rst	2020-05-27 06:53:25.000000000 +0200
+++ anaconda-33.16.3.26p/docs/intro.rst	2021-05-10 15:07:37.574090274 +0200
@@ -1,7 +1,7 @@
 Introduction to Anaconda
 ========================
 
-Anaconda is the installation program used by Fedora, Red Hat Enterprise Linux
+Anaconda is the installation program used by Fedora, EuroLinux
 and some other distributions.
 
 During installation, a target computer's hardware is identified and configured
diff -uNr anaconda-33.16.3.26/docs/iscsi.rst anaconda-33.16.3.26p/docs/iscsi.rst
--- anaconda-33.16.3.26/docs/iscsi.rst	2018-11-08 11:54:59.000000000 +0100
+++ anaconda-33.16.3.26p/docs/iscsi.rst	2021-05-10 14:43:24.457182272 +0200
@@ -139,7 +139,7 @@
 The booting problems are either due to incorrectly generated dracut boot
 arguments or they are simply dracut bugs.
 
-Note that many of the iscsi adapters are installed in different Red Hat machines
+Note that many of the iscsi adapters are installed in different EuroLinux machines
 and so the issues can often be reproduced and debugged.
 
 
diff -uNr anaconda-33.16.3.26/pyanaconda/core/constants.py anaconda-33.16.3.26p/pyanaconda/core/constants.py
--- anaconda-33.16.3.26/pyanaconda/core/constants.py	2020-07-27 15:52:47.000000000 +0200
+++ anaconda-33.16.3.26p/pyanaconda/core/constants.py	2021-05-10 14:45:53.664908434 +0200
@@ -160,8 +160,8 @@
 )
 
 WARNING_HARDWARE_UNSUPPORTED = N_(
-    "This hardware (or a combination thereof) is not supported by Red Hat. For more information "
-    "on supported hardware, please refer to http://www.redhat.com/hardware."
+    "This hardware (or a combination thereof) is not supported by EuroLinux. Because EuroLinux is RHEL"
+    "clone, please refer to https://euro-linux.com https://www.redhat.com/hardware."
 )
 
 # Storage messages
diff -uNr anaconda-33.16.3.26/pyanaconda/ui/gui/spokes/installation_source.py anaconda-33.16.3.26p/pyanaconda/ui/gui/spokes/installation_source.py
--- anaconda-33.16.3.26/pyanaconda/ui/gui/spokes/installation_source.py	2021-05-10 13:30:35.864557486 +0200
+++ anaconda-33.16.3.26p/pyanaconda/ui/gui/spokes/installation_source.py	2021-05-10 13:32:40.802388348 +0200
@@ -703,6 +703,9 @@
         self._iso_box = self.builder.get_object("isoBox")
         self._network_button = self.builder.get_object("networkRadioButton")
         self._network_box = self.builder.get_object("networkBox")
+    
+        # EuroLinux patch
+        really_hide(self._cdn_button)
 
         self._url_entry = self.builder.get_object("urlEntry")
         self._protocol_combo_box = self.builder.get_object("protocolComboBox")
diff -uNr anaconda-33.16.3.26/pyanaconda/ui/gui/spokes/subscription.py anaconda-33.16.3.26p/pyanaconda/ui/gui/spokes/subscription.py
--- anaconda-33.16.3.26/pyanaconda/ui/gui/spokes/subscription.py	2020-08-12 16:35:39.000000000 +0200
+++ anaconda-33.16.3.26p/pyanaconda/ui/gui/spokes/subscription.py	2021-05-10 14:41:03.255749272 +0200
@@ -76,10 +76,16 @@
     REGISTRATION_PAGE = 0
     SUBSCRIPTION_STATUS_PAGE = 1
 
+    # EuroLinux patch
+    @property
+    def showable(self):
+        return False
+
     @classmethod
     def should_run(cls, environment, data):
         """The Subscription spoke should run only if the Subscription module is available."""
-        return is_module_available(SUBSCRIPTION)
+        # EuroLinux patch
+        return False
 
     def __init__(self, *args):
         super().__init__(*args)
