--- /home/Alex/workspace/EuroLinux/python-rpmpatch/srpms/rhel/eth-tools.spec	2021-11-11 12:31:52.000000000 +0100
+++ /home/Alex/workspace/EuroLinux/python-rpmpatch/srpms/el/eth-tools.spec	2022-08-28 16:07:12.639395591 +0200
@@ -40,7 +40,7 @@
 Contains tools for managing fabric on a management node.
 
 %prep
-%autosetup -n eth-fast-fabric-%{version_no_tilde}
+%autosetup -n eth-fast-fabric-%{version_no_tilde} -p1
 
 %build
 cd OpenIb_Host
