diff -uNr eth-fast-fabric-11.5.0.0/build_package.sh eth-fast-fabric-11.5.0.0p/build_package.sh
--- eth-fast-fabric-11.5.0.0/build_package.sh	2023-06-01 18:57:15.000000000 +0000
+++ eth-fast-fabric-11.5.0.0p/build_package.sh	2023-11-07 15:32:48.844959090 +0000
@@ -3,7 +3,7 @@
 read id versionid <<< $(OpenIb_Host/get_id_and_versionid.sh)
 
 case $id in
-	rhel|centos|rocky|almalinux|circle|ol)
+	rhel|eurolinux|centos|rocky|almalinux|circle|ol)
 		spec_ff="OpenIb_Host/eth-tools.spec.rh"	;;
 	fedora)
 		spec_ff="OpenIb_Host/eth-tools.spec.fedora" ;;
diff -uNr eth-fast-fabric-11.5.0.0/CommonInstall/util_init.pl eth-fast-fabric-11.5.0.0p/CommonInstall/util_init.pl
--- eth-fast-fabric-11.5.0.0/CommonInstall/util_init.pl	2023-06-01 18:57:15.000000000 +0000
+++ eth-fast-fabric-11.5.0.0p/CommonInstall/util_init.pl	2023-11-07 15:33:23.567350116 +0000
@@ -245,6 +245,7 @@
 		"rhel" => "redhat",
 		"centos" => "redhat",
 		"rocky" => "redhat",
+		"eurolinux" => "redhat",
 		"almalinux" => "redhat",
 		"circle" => "redhat",
 		"oracle" => "redhat",
@@ -257,6 +258,7 @@
 		"rhel" => $NETWORK_CONF_DIR,
 		"centos" => $NETWORK_CONF_DIR,
 		"rocky" => $NETWORK_CONF_DIR,
+		"eurolinux" => $NETWORK_CONF_DIR,
 		"almalinux" => $NETWORK_CONF_DIR,
 		"circle" => $NETWORK_CONF_DIR,
 		"oracle" => $NETWORK_CONF_DIR,
diff -uNr eth-fast-fabric-11.5.0.0/MakeTools/funcs-ext.sh eth-fast-fabric-11.5.0.0p/MakeTools/funcs-ext.sh
--- eth-fast-fabric-11.5.0.0/MakeTools/funcs-ext.sh	2023-06-01 18:57:15.000000000 +0000
+++ eth-fast-fabric-11.5.0.0p/MakeTools/funcs-ext.sh	2023-11-07 15:33:11.450222146 +0000
@@ -738,6 +738,7 @@
         [fedora]="redhat"
         [rocky]="redhat"
         [almalinux]="redhat"
+        [eurolinux]="redhat"
         [circle]="redhat"
         [ol]="redhat"
         [opencloudos]="opencloudos"
diff -uNr eth-fast-fabric-11.5.0.0/OpenIb_Host/get_id_and_versionid.sh eth-fast-fabric-11.5.0.0p/OpenIb_Host/get_id_and_versionid.sh
--- eth-fast-fabric-11.5.0.0/OpenIb_Host/get_id_and_versionid.sh	2023-06-01 18:57:15.000000000 +0000
+++ eth-fast-fabric-11.5.0.0p/OpenIb_Host/get_id_and_versionid.sh	2023-11-07 15:31:58.308370999 +0000
@@ -12,6 +12,7 @@
     [rhel]="rhel"
     [centos]="rhel"
     [rocky]="rhel"
+    [eurolinux]="rhel"
     [almalinux]="rhel"
     [circle]="rhel"
     [ol]="ol"
