diff --git a/postgis34.spec b/postgis34.spec
index 828252f..080533b 100644
--- a/postgis34.spec
+++ b/postgis34.spec
@@ -210,7 +210,7 @@ export PKG_CONFIG_PATH=$PKG_CONFIG_PATH:%{projinstdir}/lib64/pkgconfig

 autoconf

-%configure --with-pgconfig=%{pginstdir}/bin/pg_config \
+%configure CC="gcc" --with-pgconfig=%{pginstdir}/bin/pg_config \
        --bindir=%{pginstdir}/bin/ \
        --datadir=%{pginstdir}/share/ \
        --enable-lto \
