--- /home/Alex/workspace/EuroLinux/python-rpmpatch/srpms/rhel/gnome-settings-daemon.spec	2022-02-14 16:48:41.000000000 +0100
+++ /home/Alex/workspace/EuroLinux/python-rpmpatch/srpms/el/gnome-settings-daemon.spec	2022-08-28 22:16:56.121031998 +0200
@@ -171,12 +171,6 @@
 %{_libexecdir}/gsd-wwan
 %{_datadir}/glib-2.0/schemas/org.gnome.settings-daemon.plugins.wwan.gschema.xml
 
-%{_sysconfdir}/xdg/autostart/org.gnome.SettingsDaemon.Subscription.desktop
-%{_libexecdir}/gsd-subman
-%{_libexecdir}/gsd-subman-helper
-%{_datadir}/polkit-1/actions/org.gnome.settings-daemon.plugins.subman.policy
-%{_datadir}/polkit-1/rules.d/org.gnome.settings-daemon.plugins.subman.rules
-
 %dir %{_libdir}/gnome-settings-daemon-40
 %{_libdir}/gnome-settings-daemon-40/libgsd.so
 
