From e6f785197f8570aea622528eed7572f2ba5d97f9 Mon Sep 17 00:00:00 2001
From: Pat Riehecky <riehecky@fnal.gov>
Date: Tue, 8 Jul 2014 13:47:19 -0500
Subject: [LIBREPORT PATCH 9992/2] Don't try and open any support cases upstream

---
 src/plugins/rhtsupport_event.conf | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/plugins/rhtsupport_event.conf b/src/plugins/rhtsupport_event.conf
index 58261bb..e803686 100644
--- a/src/plugins/rhtsupport_event.conf
+++ b/src/plugins/rhtsupport_event.conf
@@ -1,7 +0,0 @@
-EVENT=report_RHTSupport
-    # Submit an uReport and create a case in Red Hat Customer Portal
-    reporter-rhtsupport -u
-
-EVENT=report_RHTSupport_AddData
-    # Attach data to a case in Red Hat Customer Portal
-    reporter-rhtsupport -t
-- 
1.8.3.1

