From c2148ad44b72ed45bcf835ae29f3415ab7856590 Mon Sep 17 00:00:00 2001
From: rpm-build <rpm-build>
Date: Wed, 23 Aug 2017 10:19:24 -0500
Subject: [PATCH] Don't report anaconda errors to TUV, not their product

---
 src/workflows/anaconda_event.conf | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/src/workflows/anaconda_event.conf b/src/workflows/anaconda_event.conf
index e894a53..41bc2f8 100644
--- a/src/workflows/anaconda_event.conf
+++ b/src/workflows/anaconda_event.conf
@@ -1,4 +1 @@
-EVENT=workflow_AnacondaFedora component=anaconda
-EVENT=workflow_AnacondaRHEL component=anaconda
-EVENT=workflow_AnacondaRHELBugzilla component=anaconda release_type=pre-release
 EVENT=workflow_AnacondaUpload component=anaconda
-- 
1.8.3.1

