--- libreport.spec.orig	2018-04-19 12:52:57.810277457 -0500
+++ libreport.spec	2018-04-19 12:55:40.220855221 -0500
@@ -292,6 +292,12 @@ Requires: libreport-filesystem = %{versi
 Requires: libreport-python = %{version}-%{release}
 Requires: satyr >= %{satyr_ver}
 
+# For Scientific Linux to avoid implying we have RHEL support
+# and clean up older releases where this was missed
+Obsoletes: libreport-rhel < 2.1.11-24
+Obsoletes: libreport-rhel-anaconda-bugzilla < 2.1.11-24
+Obsoletes: libreport-rhel-bugzilla < 2.1.11-24
+Obsoletes: libreport-plugin-rhtsupport < 2.1.11-24
 
 # for rhel6
 %if 0%{?rhel} == 6
@@ -346,9 +352,9 @@ Provides: report = 0:0.23-1
 Obsoletes: report < 0:0.23-1
 # in report the rhtsupport is in the main package, so we need to install it too
 # report is only in RHEL6, we do not need to carry the dependency to newer RHELs
-%if 0%{?rhel} == 6
-Requires: libreport-plugin-rhtsupport
-%endif
+#%if 0%{?rhel} == 6
+#Requires: libreport-plugin-rhtsupport
+#%endif
 
 %description python
 Python bindings for report-libs.
@@ -458,8 +464,9 @@ Plugin to report bugs into RH support sy
 Summary: %{name}'s compat layer for obsoleted 'report' package
 Group: System Environment/Libraries
 Requires: libreport = %{version}-%{release}
-Requires: %{name}-plugin-bugzilla = %{version}-%{release}
-Requires: %{name}-plugin-rhtsupport = %{version}-%{release}
+Requires: %{name}-plugin-reportuploader = %{version}-%{release}
+#Requires: %{name}-plugin-bugzilla = %{version}-%{release}
+#Requires: %{name}-plugin-rhtsupport = %{version}-%{release}
 
 %description compat
 Provides 'report' command-line tool.
@@ -526,10 +533,10 @@ Group: Applications/File
 Requires: %{name} = %{version}-%{release}
 Requires: libreport-plugin-reportuploader = %{version}-%{release}
 # The line below should be removed in RHEL7 GA
-Requires: libreport-plugin-bugzilla = %{version}-%{release}
-%if 0%{?rhel}
-Requires: libreport-plugin-rhtsupport = %{version}-%{release}
-%endif
+#Requires: libreport-plugin-bugzilla = %{version}-%{release}
+#%if 0%{?rhel}
+#Requires: libreport-plugin-rhtsupport = %{version}-%{release}
+#%endif
 
 %description anaconda
 Default configuration for reporting Anaconda problems or uploading the gathered
