From b29c1af9ab7713f2b206fd4adec3dab44243ac18 Mon Sep 17 00:00:00 2001
From: Alex Baranowski <alex@euro-linux.com>
Date: Sun, 9 Apr 2023 21:21:18 +0200
Subject: [PATCH] eurolinux patches

---
 MakeTools/funcs-ext.sh | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/MakeTools/funcs-ext.sh b/MakeTools/funcs-ext.sh
index 16683d8..bcd71e0 100755
--- a/MakeTools/funcs-ext.sh
+++ b/MakeTools/funcs-ext.sh
@@ -725,6 +725,9 @@ function os_vendor()
             centos)
                 rval=redhat
                 ;;
+            eurolinux)
+                rval=redhat
+                ;;
             rocky)
                 rval=redhat
                 ;;
@@ -757,6 +760,9 @@ function os_vendor()
 				rval=UnitedLinux
 			fi
 		elif [ $rval = 'centos' ]
+		then
+			rval=redhat
+		elif [ $rval = 'eurolinux' ]
 		then
 			rval=redhat
 		elif [ $rval = 'rocky' ]
@@ -836,6 +842,10 @@ function os_vendor_version()
 		then
 			# CentOS 
 			rval="ES"`cat /etc/redhat-release | sed -r 's/^.+([[:digit:]])\.([[:digit:]]).+$/\1\2/'`
+		elif grep -qi eurolinux /etc/redhat-release
+		then
+			# EuroLinux 
+			rval="ES"`cat /etc/redhat-release | sed -r 's/^.+([[:digit:]])\.([[:digit:]]).+$/\1\2/'`
 		elif grep -qi rocky /etc/redhat-release
 		then
 			# Rocky
-- 
2.31.1

