diff -uNr cloud-init-19.4/cloudinit/config/cc_ntp.py cloud-init-19.4p/cloudinit/config/cc_ntp.py
--- cloud-init-19.4/cloudinit/config/cc_ntp.py  2019-12-18 15:14:55.000000000 +0000
+++ cloud-init-19.4p/cloudinit/config/cc_ntp.py 2020-10-27 14:35:03.058363638 +0000
@@ -25,7 +25,7 @@
 frequency = PER_INSTANCE
 NTP_CONF = '/etc/ntp.conf'
 NR_POOL_SERVERS = 4
-distros = ['centos', 'debian', 'fedora', 'opensuse', 'rhel', 'sles', 'ubuntu']
+distros = ['centos', 'debian', 'eurolinux', 'fedora', 'opensuse', 'rhel', 'sles', 'ubuntu']
 
 NTP_CLIENT_CONFIG = {
     'chrony': {
@@ -369,6 +369,8 @@
     # use the opensuse pool
     if distro == 'sles':
         pool_distro = 'opensuse'
+    if distro == 'eurolinux':
+        pool_distro = ''
     for x in range(0, NR_POOL_SERVERS):
         name = "%d.%s.pool.ntp.org" % (x, pool_distro)
         names.append(name)
diff -uNr cloud-init-19.4/cloudinit/config/cc_yum_add_repo.py cloud-init-19.4p/cloudinit/config/cc_yum_add_repo.py
--- cloud-init-19.4/cloudinit/config/cc_yum_add_repo.py 2019-12-18 15:14:55.000000000 +0000
+++ cloud-init-19.4p/cloudinit/config/cc_yum_add_repo.py  2020-10-27 14:35:24.741029541 +0000
@@ -40,7 +40,7 @@
 
 from cloudinit import util
 
-distros = ['fedora', 'rhel']
+distros = ['fedora', 'rhel', 'eurolinux']
 
 
 def _canonicalize_id(repo_id):
diff -uNr cloud-init-19.4/cloudinit/distros/eurolinux.py cloud-init-19.4p/cloudinit/distros/eurolinux.py
--- cloud-init-19.4/cloudinit/distros/eurolinux.py  1970-01-01 00:00:00.000000000 +0000
+++ cloud-init-19.4p/cloudinit/distros/eurolinux.py 2020-10-27 14:36:00.573657130 +0000
@@ -0,0 +1,9 @@
+# This file is part of cloud-init. See LICENSE file for license information.
+
+from cloudinit.distros import rhel
+
+
+class Distro(rhel.Distro):
+        pass
+
+# vi: ts=4 expandtab
diff -uNr cloud-init-19.4/cloudinit/distros/__init__.py cloud-init-19.4p/cloudinit/distros/__init__.py
--- cloud-init-19.4/cloudinit/distros/__init__.py 2019-12-18 15:14:55.000000000 +0000
+++ cloud-init-19.4p/cloudinit/distros/__init__.py  2020-10-27 14:35:44.634723009 +0000
@@ -36,7 +36,7 @@
 
 OSFAMILIES = {
     'debian': ['debian', 'ubuntu'],
-    'redhat': ['amazon', 'centos', 'fedora', 'rhel'],
+    'redhat': ['amazon', 'centos', 'fedora', 'rhel', 'eurolinux'],
     'gentoo': ['gentoo'],
     'freebsd': ['freebsd'],
     'suse': ['opensuse', 'sles'],
diff -uNr cloud-init-19.4/cloudinit/net/sysconfig.py cloud-init-19.4p/cloudinit/net/sysconfig.py
--- cloud-init-19.4/cloudinit/net/sysconfig.py  2020-10-27 14:33:03.885514957 +0000
+++ cloud-init-19.4p/cloudinit/net/sysconfig.py 2020-10-27 14:36:25.829747531 +0000
@@ -18,7 +18,7 @@
 
 LOG = logging.getLogger(__name__)
 NM_CFG_FILE = "/etc/NetworkManager/NetworkManager.conf"
-KNOWN_DISTROS = ['centos', 'fedora', 'rhel', 'suse']
+KNOWN_DISTROS = ['centos', 'eurolinux', 'fedora', 'rhel', 'suse']
 
 
 def _make_header(sep='#'):
diff -uNr cloud-init-19.4/cloudinit/util.py cloud-init-19.4p/cloudinit/util.py
--- cloud-init-19.4/cloudinit/util.py 2020-10-27 14:33:03.952515183 +0000
+++ cloud-init-19.4p/cloudinit/util.py  2020-10-27 14:37:08.952902654 +0000
@@ -668,7 +668,7 @@
     if system == "linux":
         linux_dist = info['dist'][0].lower()
         if linux_dist in (
-                'arch', 'centos', 'debian', 'fedora', 'rhel', 'suse'):
+                'arch', 'centos', 'debian', 'fedora', 'rhel', 'suse', 'eurolinux'):
             var = linux_dist
         elif linux_dist in ('ubuntu', 'linuxmint', 'mint'):
             var = 'ubuntu'
diff -uNr cloud-init-19.4/config/cloud.cfg.tmpl cloud-init-19.4p/config/cloud.cfg.tmpl
--- cloud-init-19.4/config/cloud.cfg.tmpl 2019-12-18 15:14:55.000000000 +0000
+++ cloud-init-19.4p/config/cloud.cfg.tmpl  2020-10-27 14:37:54.164065286 +0000
@@ -21,7 +21,7 @@
 disable_root: true
 {% endif %}
 
-{% if variant in ["centos", "fedora", "rhel"] %}
+{% if variant in ["centos", "fedora", "rhel", "eurolinux"] %}
 mount_default_fields: [~, ~, 'auto', 'defaults,nofail', '0', '2']
 resize_rootfs_tmp: /dev
 ssh_pwauth:   0
@@ -137,7 +137,7 @@
 # (not accessible to handlers/transforms)
 system_info:
    # This will affect which distro class gets used
-{% if variant in ["arch", "centos", "debian", "fedora", "freebsd", "rhel", "suse", "ubuntu"] %}
+{% if variant in ["arch", "centos", "debian", "fedora", "freebsd", "rhel", "suse", "ubuntu", "eurolinux"] %}
    distro: {{ variant }}
 {% else %}
    # Unknown/fallback distro.
@@ -185,7 +185,7 @@
          primary: http://ports.ubuntu.com/ubuntu-ports
          security: http://ports.ubuntu.com/ubuntu-ports
    ssh_svcname: ssh
-{% elif variant in ["arch", "centos", "fedora", "rhel", "suse"] %}
+{% elif variant in ["arch", "centos", "eurolinux",  "fedora", "rhel", "suse"] %}
    # Default user name + that default users groups (if added/used)
    default_user:
      name: {{ variant }}
Binary files cloud-init-19.4/.git/index and cloud-init-19.4p/.git/index differ
diff -uNr cloud-init-19.4/systemd/cloud-init-generator.tmpl cloud-init-19.4p/systemd/cloud-init-generator.tmpl
--- cloud-init-19.4/systemd/cloud-init-generator.tmpl 2020-10-27 14:33:03.899515004 +0000
+++ cloud-init-19.4p/systemd/cloud-init-generator.tmpl  2020-10-27 14:38:28.405188458 +0000
@@ -83,7 +83,7 @@
 
 check_for_datasource() {
     local ds_rc=""
-{% if variant in ["fedora", "centos"] %}
+{% if variant in ["fedora", "centos", "eurolinux"] %}
     local dsidentify="/usr/libexec/cloud-init/ds-identify"
 {% else %}
     local dsidentify="/usr/lib/cloud-init/ds-identify"
diff -uNr cloud-init-19.4/tools/read-dependencies cloud-init-19.4p/tools/read-dependencies
--- cloud-init-19.4/tools/read-dependencies 2019-12-18 15:14:55.000000000 +0000
+++ cloud-init-19.4p/tools/read-dependencies  2020-10-27 14:39:28.703405362 +0000
@@ -23,6 +23,7 @@
 # Map the appropriate package dir needed for each distro choice
 DISTRO_PKG_TYPE_MAP = {
     'centos': 'redhat',
+    'eurolinux': 'redhat',
     'redhat': 'redhat',
     'debian': 'debian',
     'ubuntu': 'debian',
@@ -57,11 +58,13 @@
 
 DRY_DISTRO_INSTALL_PKG_CMD = {
     'centos': ['yum', 'install', '--assumeyes'],
+    'eurolinux': ['yum', 'install', '--assumeyes'],
     'redhat': ['yum', 'install', '--assumeyes'],
 }
 
 DISTRO_INSTALL_PKG_CMD = {
     'centos': MAYBE_RELIABLE_YUM_INSTALL,
+    'eurolinux': MAYBE_RELIABLE_YUM_INSTALL,
     'redhat': MAYBE_RELIABLE_YUM_INSTALL,
     'debian': ['apt', 'install', '-y'],
     'ubuntu': ['apt', 'install', '-y'],
@@ -75,6 +78,7 @@
     'common': ['make', 'sudo', 'tar'],
     'redhat': ['python-tox'],
     'centos': ['python-tox'],
+    'eurolinux': ['python-tox'],
     'ubuntu': ['devscripts', 'python3-dev', 'libssl-dev', 'tox', 'sbuild'],
     'debian': ['devscripts', 'python3-dev', 'libssl-dev', 'tox', 'sbuild']}
 
@@ -258,7 +262,7 @@
         cmd = DRY_DISTRO_INSTALL_PKG_CMD[distro]
     install_cmd.extend(cmd)
 
-    if distro in ['centos', 'redhat']:
+    if distro in ['centos', 'redhat', 'eurolinux']:
         # CentOS and Redhat need epel-release to access oauthlib and jsonschema
         subprocess.check_call(install_cmd + ['epel-release'])
     if distro in ['suse', 'opensuse', 'redhat', 'centos']:
diff -uNr cloud-init-19.4/tools/render-cloudcfg cloud-init-19.4p/tools/render-cloudcfg
--- cloud-init-19.4/tools/render-cloudcfg 2019-12-18 15:14:55.000000000 +0000
+++ cloud-init-19.4p/tools/render-cloudcfg  2020-10-27 14:39:45.935467348 +0000
@@ -4,7 +4,7 @@
 import os
 import sys
 
-VARIANTS = ["arch", "centos", "debian", "fedora", "freebsd", "rhel", "suse",
+VARIANTS = ["arch", "centos", "eurolinux", "debian", "fedora", "freebsd", "rhel", "suse",
             "ubuntu", "unknown"]
 
 if "avoid-pep8-E402-import-not-top-of-file":
