diff -uNr Python-3.6.8-oryginal/Lib/platform.py Python-3.6.8/Lib/platform.py
--- Python-3.6.8-oryginal/Lib/platform.py        2021-04-29 15:06:02.226430252 +0000
+++ Python-3.6.8/Lib/platform.py        2021-04-29 15:36:42.845110517 +0000
@@ -299,7 +299,7 @@
 _supported_dists = (
     'SuSE', 'debian', 'fedora', 'redhat', 'centos',
     'mandrake', 'mandriva', 'rocks', 'slackware', 'yellowdog', 'gentoo',
-    'UnitedLinux', 'turbolinux', 'arch', 'mageia')
+    'UnitedLinux', 'turbolinux', 'arch', 'mageia', 'eurolinux')
 
 def _parse_release_file(firstline):
